packages/trpc/server/routers/viewer/workflows/update.handler.ts

```diff
@@ -257,13 +257,13 @@ export const updateHandler = async ({ ctx, input }: UpdateOptions) => {
         },
       });
 
-      steps.forEach(async (step) => {
+      const pSteps = steps.map(async (step) => {
```

Comment:
```suggestion
      const promiseSteps = steps.map(async (step) => {
```
Let's be clear here.